pkg/spanconfig/spanconfigtestutils/utils.go

```
@@ -192,12 +248,21 @@ func PrintSpan(sp roachpb.Span) string {
 	return fmt.Sprintf("[%s,%s)", string(sp.Key), string(sp.EndKey))
 }
 
+func printSystemTarget(target spanconfig.SystemTarget) string {
+	return fmt.Sprintf("{source=%s,target=%s}", target.SourceTenantID, target.TargetTenantID)
+}
+
 // PrintTarget is a helper function that prints a spanconfig.Target.
-func PrintTarget(target spanconfig.Target) string {
-	if target.IsSpanTarget() {
+func PrintTarget(t *testing.T, target spanconfig.Target) string {
+	switch {
+	case target.IsSpanTarget():
 		return PrintSpan(target.GetSpan())
+	case target.IsSystemTarget():
+		return printSystemTarget(target.GetSystemTarget())
+	default:
+		t.Fatalf("unknown traget type")
```

Typo: traget. Could panic below instead of the empty string return.